release.yml

```diff
@@ -166,18 +163,17 @@ stages:
     #   displayName: test push
 
     - script: |
-        dotnet test **/*FunctionalTests.dll
+        dotnet test FhirConverterBuild/**/*FunctionalTests.dll
       displayName: Functional Tests
     - task: Docker@2
       displayName: stop registry
       condition: succeededOrFailed()
       inputs:
         command: stop
         container: registry
-
-  - job: Linux_Functional_Test
-    pool:
-      vmImage: 'ubuntu-latest'
+  
+  - job: OSX_Functional_Test
```

Comment:
Are we removing the Linux tests here?

Reply:
We can add it later when supporting linux.